buildbot/check.py

```diff
@@ -36,6 +69,10 @@ def main():
     parser.add_argument("-s", "--src-dir", metavar="SRC_DIR", help="source directory")
     parser.add_argument("-o", "--obj-dir", metavar="OBJ_DIR", required=True, help="build directory")
     parser.add_argument("-t", "--test-suite", metavar="TEST_SUITE", required=True, help="check-xxx target")
+    parser.add_argument("-u", "--unittests-suite", metavar="UNITTESTS_SUITE", default="check-unittests-sycl",
+                        help="check-unittests-sycl-xxx target")
```

Comment:
I am unsure how to enable the automatic buildbot runs to add which unittest target to run, therefore providing as a default the phony target that runs all SYCL unittests.

Reply:
Thanks to @AlexeySachkov I figured out how LIT integrates with the unittests which makes this change unnecessary and I will remove it.